ng-dev/release/stamping/env-stamp.ts

```diff
@@ -51,40 +48,39 @@ function hasLocalChanges(git: GitClient) {
  * Get the versions for generated packages.
  *
  * In snapshot mode, the version is based on the most recent semver tag.
- * In release mode, the version is based on the base package.json version.
+ * In release mode, the version is based on the workspace version.
  */
 function getSCMVersions(
   git: GitClient,
   mode: EnvStampMode,
 ): {version: string; experimentalVersion: string} {
-  if (mode === 'snapshot') {
-    const localChanges = hasLocalChanges(git) ? '.with-local-changes' : '';
-    const {stdout: rawVersion} = git.run([
-      'describe',
-      '--match',
-      // As git describe uses glob matchers we cannot the specific describe what we expect to see
-      // starting character we expect for our version string.  To ensure we can handle 'v'
-      // prefixed verstions we have the '?' wildcard character.
-      '?[0-9]*.[0-9]*.[0-9]*',
-      '--abbrev=7',
-      '--tags',
-      'HEAD',
-    ]);
-    const {version} = new SemVer(rawVersion);
-    const {version: experimentalVersion} = createExperimentalSemver(version);
+  if (mode === 'release') {
+    const workspaceVersion = getVersionFromWorkspacePackageJson(git);
```

Comment:
This diff here is quite a lot, but only the `mode === 'release'` case got updated, and I swaped the blocks for better readability. Feel free to ignore the rest.